dynamodb-stream-eventbridge-fanout/src/main/java/no/unit/nva/doi/utils/JacocoGenerated.java

```diff
@@ -0,0 +1,4 @@
+package no.unit.nva.doi.utils;
+
+public @interface JacocoGenerated {
```

Comment:
Why can't we  use commons for now and in the near feature we split commons? I think it will be easier to just split commons than to remove replicated code later.